dbmongo/js/reduce.algo2/fraisFinancier.js

```diff
@@ -1,15 +1,32 @@
 exports.__esModule = true;
+exports.fraisFinancier = void 0;
 function fraisFinancier(diane) {
     "use strict";
-    if (("interets" in diane) && (diane["interets"] !== null) &&
-        ("excedent_brut_d_exploitation" in diane) && (diane["excedent_brut_d_exploitation"] !== null) &&
-        ("produits_financiers" in diane) && (diane["produits_financiers"] !== null) &&
-        ("charges_financieres" in diane) && (diane["charges_financieres"] !== null) &&
-        ("charge_exceptionnelle" in diane) && (diane["charge_exceptionnelle"] !== null) &&
-        ("produit_exceptionnel" in diane) && (diane["produit_exceptionnel"] !== null) &&
-        diane["excedent_brut_d_exploitation"] + diane["produits_financiers"] + diane["produit_exceptionnel"] - diane["charge_exceptionnelle"] - diane["charges_financieres"] != 0) {
-        return diane["interets"] / (diane["excedent_brut_d_exploitation"] + diane["produits_financiers"] + diane["produit_exceptionnel"] -
-            diane["charge_exceptionnelle"] - diane["charges_financieres"]) * 100;
+    if ("interets" in diane &&
+        diane["interets"] !== null &&
+        "excedent_brut_d_exploitation" in diane &&
+        diane["excedent_brut_d_exploitation"] !== null &&
+        "produits_financiers" in diane &&
+        diane["produits_financiers"] !== null &&
+        "charges_financieres" in diane &&
+        diane["charges_financieres"] !== null &&
+        "charge_exceptionnelle" in diane &&
+        diane["charge_exceptionnelle"] !== null &&
+        "produit_exceptionnel" in diane &&
+        diane["produit_exceptionnel"] !== null &&
+        diane["excedent_brut_d_exploitation"] +
+            diane["produits_financiers"] +
+            diane["produit_exceptionnel"] -
+            diane["charge_exceptionnelle"] -
+            diane["charges_financieres"] !=
```

Comment:
Question: l'usage de l'opérateur `!=` (au lieu de son alternative stricte `!==`) fait râler codacy: https://app.codacy.com/gh/signaux-faibles/opensignauxfaibles/pullRequest?prid=5511987

=> Peut-on le remplacer par `!==` ?

Reply:
Oui je pense, on a bien le même type des deux côtés de l'inégalité (à changer dans le fichier .ts j'imagine)

Reply:
fait dans https://github.com/signaux-faibles/opensignauxfaibles/pull/46/commits/af890c758506469acab25f5d129dc1b72ae61b2f